### PR TITLE
upgraded the dropwizard version to 0.8.1, also update the gradle bint…

### PR DESCRIPTION
**Pull request by pandey-vikesh**
we are using dropwizard 0.8.1 with mongodb and were looking for a dropwizard plugin for it. I tried integrating this plugin but it failed as my code was built on 0.8.1 dropwizard but this library/jar was built on 0.7.1. So i though of upgrading it and tested it by manually supplying the upgraded jar in my project and it worked fine. Could you please merge this change and publish it to bintray with a new version so that we can use it seamlessly in our project.

### CODE REVIEW

**Comment by mhurne** [comment]
Thanks for the pull request! Is there any particular reason why this pull request needs to include an upgrade to gradle-bintray-plugin?

**Comment by pandey-vikesh** [comment]
Hi Matt,

Apologies for the delayed reply. commit c9d419a6ff180f9f1766fc2a96856a16312645fc is specifically to upgrade the dropwizard library from 0.7.1 to 0.8.1. 
Reason: The current dropwizard-mongeez library is not compatible with 0.8.x dropwizard projects. As i have informed in my earlier comment, we are running a dropwizard 0.8.x project and wanted to use this library to migrate our mongo db. but we could not, as the migration process failed due a class being moved to a different package while upgrading the dropwizard from 0.7.1 to 0.8.1. Hence the upgrade is required. Hope it answers what you are asking. If not, please skype me : vikesh.pandey.18 and we can discuss it over a quick call :)

Thanks in Advance,
Vikesh

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ logs/
 out/
 .DS_Store
 .gradletasknamecache
+/bin/
```
Comment (mhurne) [inline .gitignore]: What is the reason for this addition?

File: build.gradle [hunk 2]
```diff
@@ -1,10 +1,14 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+      url "https://plugins.gradle.org/m2/"
```
Comment (mhurne) [inline build.gradle]: What is the reason for this addition?

File: build.gradle [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+    }
+        
     }
     dependencies {
         classpath "com.github.townsfolk:gradle-release:1.2"
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:0.5"
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.2"
```
Comment (mhurne) [inline build.gradle]: Is this change necessary to upgrade to Dropwizard 0.8.1?

File: build.gradle [hunk 4]
```diff
@@ -12,14 +16,15 @@ apply plugin: "java"
 apply plugin: "groovy"
 apply plugin: "maven"
 apply plugin: "maven-publish"
+apply plugin: "com.jfrog.bintray"
```
Comment (mhurne) [inline build.gradle]: Is this change necessary to upgrade to Dropwizard 0.8.1?

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mhurne) [inline pom.xml]: This project uses Gradle rather than Maven. Unless I'm missing something, please remove this file from the merge request.

Comment (pandey-vikesh) [inline pom.xml]: yeah sure, apologies for the clutter. Bintray upgrade was required as it
was not building with upgraded dropwizard version. I was on vacation for
past couple weeks, could not reply on this thread on time. I believe it
would be nicer if we can have a skype discussion to discuss why i did what
i did. Let me know if this suits you, if yes, please do share your skype id
and we can block some time to discuss the forked changes.


Thanks & Regards,
Vikesh
+91-9958130911

Comment (mhurne) [inline pom.xml]: I don't think a Skype call should be necessary.

**Comment by mhurne** [comment]
Closing due to inactivity.
